### ISSUE TITLE
Check alt text on all images

### ISSUE THREAD

**Issue by cthomasONRR** [ONRR/onrr.gov-site#2047]
I need to go though every page within the CMS and make sure that each instance of each image has alt text in it. We previously believed that having the alt text in the description field of an image file would carry over to the usage of an image on a page, so we wouldn't have to manually add the alt text at each instance we use the same image. But it seems we do. This issue is for me to make sure that the alt text is in all locations of an image.

**Comment by cthomasONRR** [comment]
@mcharg I went through all webpages/CMS pages and made sure every image had alt text in each page. All alt text issues should be resolved now. Would you mind spot-checking?

**Comment by mcharg** [comment]
@cthomasONRR I did some spot check and running lighthouse audits on pages with lots of images and not finding any with missing alt text.
